README.md

```diff
@@ -14,6 +14,24 @@ kubectl apply -f https://raw.githubusercontent.com/juicedata/juicefs-csi-driver/
 
 Additional steps could be required on some provider, e.g. Aliyun Container Service Kubernetes. See [Troubleshooting#AttachVolume.Attach failed](docs/DEVELOP.md#attachvolumeattach-failed) for details.
 
+## Upgrade
+
+We have two components to upgrade:
+
+* CSI Driver
+* JuiceFS client in CSI Driver
+
+### Upgrade CSI Driver
+
+1. Stop all pods using this driver.
+2. Upgrade driver:
+	* If you're using `latest` tag, simple run `kubectl apply -f` like [installation](#installation).
```

Comment:
This command seems incomplete?

Besides, if there is nothing changed in `k8s.yaml`, no upgrade will be triggered IIUC. Try something like `kubectl rollout ...`?

Reply:
updated